allennlp/tests/modules/token_embedders/elmo_token_embedder_test.py

```diff
@@ -87,10 +87,12 @@ def test_forward_works_with_projection_layer(self):
         word2[2] = 1
         word2[3] = 0
         embedding_layer = ElmoTokenEmbedder.from_params(vocab=None, params=params)
+        assert embedding_layer.get_output_dim() == 20
+        
         input_tensor = torch.LongTensor([[word1, word2]])
         embedded = embedding_layer(input_tensor).data.numpy()
         assert embedded.shape == (1, 2, 20)
-
+        
```

Comment:
likewise, looks like there's some whitespace on this line that pylint doesn't like.